vertical-pod-autoscaler/enhancements/4016-in-place-updates-support/README.md

```diff
@@ -254,6 +257,48 @@ tested in the following scenarios:
 * In-place update will fail but `CanEvict` is false, pod should not be evicted.
 * In-place update will fail but `EvictionRequirements` are false, pod should not be evicted.
 
+### Upgrade / Downgrade Strategy
+
+#### Upgrade
+
+On upgrade of the VPA to 1.4.0 (tentative release version), nothing will change,
+VPAs will continue to work as before.
+
+Users can use the new `InPlaceOrRecreate` by enabling the alpha Feature Gate (which defaults to disabled)
+by passing `--feature-gates=InPlaceVerticalScaling=true` to the updater and admission-controller components and setting
+their VPA UpdateMode to use `InPlaceOrRecreate`.
+
+#### Downgrade
+
+On downgrade of VPA from 1.4.0 (tentative release version), nothing will change.
+VPAs will continue to work as previously, unless, the user had enabled the feature
+gate. In which case downgrade could break their VPA that uses `InPlaceOrRecreate`.
+
+### Feature Enablement and Rollback
+
+#### How can this feature be enabled / disabled in a live cluster?
+
+- Feature gate name: InPlaceVerticalScaling
```

Comment:
Are we okay with that feature-gate name? I put it in my PR, but I know there was the whole discussion on Slack about the KEP `InPlacePodVerticalScaling` name and how it was confused with the VPA, so just want to make sure from the community's perspective that we're good with this.

Reply:
My biggest worry is the name of the feature in k/k vs in the VPA. Since Kubernetes doesn't provide an autoscaling mechanism, but rather the underlying "API" to resize pods.
I'm OK with this name, but I'm wondering if `InPlaceResize` may also make sense?
The VPA only does vertical scaling, so it's implied that a resize would be vertical. 

I'm not too fussed with it here, since the VPA is the thing that does the Vertical Scaling.

Reply:
I like it, updated.